README.md

```diff
@@ -36,12 +37,35 @@ Final render:
       <h1>Hello world</h1>
     </div>
 
+## React Native
+
 To activate React Native support you must pass in a `native` plugin option like so:
 
     plugins: [
       ["@fullstory/babel-plugin-annotate-react", { native: true }]
     ]
 
+### React Native with FullStory 
+
+When using this library with [FullStory for Mobile Apps](https://www.fullstory.com/platform/mobile-apps/), we recommend setting the `componentAttribute` and `elementAttribute` options both to `'fsTagName'` to generate better privacy selectors. 
+
+⚠️ Important: Existing FullStory privacy selectors and defined elements may need to be updated if the app was previously published without `componentAttribute` and `elementAttribute` set.
+<!-- todo: write up a KB article to walk customers through transitioning to `fsTagName` if they have pre-existing privacy selectors or defined elements; link to it here -->
+
+    plugins: [
+      '@fullstory/react-native',
+      ["@fullstory/annotate-react", {
```

Comment:
Should this be "@fullstory/annotate-react" or "@fullstory/babel-plugin-annotate-react"? It seems to work either way; maybe we should use the longer name for clarity?

```suggestion
      ["@fullstory/babel-plugin-annotate-react", {
```

Reply:
Yeah, I'd lean towards the long version, but I don't feel very strongly about it. Which one do you think is more idiomatic?

Reply:
From babel's [docs](https://babeljs.io/docs/options#name-normalization) it's hinted that they added "name normalization" specifically so we can avoid prefixing `babel-plugin` every time. In their own docs they lean towards excluding the prefix.

I vote leaving off the prefix.